src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -891,24 +851,24 @@ coverage:
 	@ echo Running 'litani build'
 	$(LITANI) run-build
 
-# Choose the invocation of cbmc-viewer depending on which version of
-# cbmc-viewer is installed.  The --version flag is not implemented in
-# version 1 --- it is an "unrecognized argument" --- but it is
-# implemented in version 2.
-_report1: $(PROOFDIR)/html
-_report2: $(PROOFDIR)/report
-_report:
-	(cbmc-viewer --version 2>&1 | grep "unrecognized argument" > /dev/null) && \
-		$(MAKE) -B _report1 || $(MAKE) -B _report2
-
-report report1 report2:
+_report: $(PROOFDIR)/report
+report:
 	@ echo Running 'litani init'
 	$(LITANI) init --project $(PROJECT_NAME)
 	@ echo Running 'litani add-job'
 	$(MAKE) -B _report
 	@ echo Running 'litani build'
 	$(LITANI) run-build
 
+_report_no_coverage: $(PROOFDIR)/report-no-coverage
+report-no-coverage:
```

Comment:
Could add:

`* make report-no-coverage: builds the final report without calculating coverage` after L17

inside of `src/cbmc_starter_kit/template-for-repository/proofs/README.md`.

Reply:
Done.  Actually rewrite the text to be more readabale.